js/company.js

```diff
@@ -0,0 +1,90 @@
+import {fetchData} from "./utils.js"
+const urlParams = new URLSearchParams(location.search);
+const symbol = urlParams.get("symbol");
+const companyIndustry = document.getElementById("companyIndustry");
+const companyFullName = document.getElementById("companyName");
+const stockPrice = document.getElementById("stockPrice");
+const stockPriceChange = document.getElementById("stockPriceChange");
+const companyDescription = document.getElementById("companyDescription");
+const companyLogo = document.getElementById("companyLogo");
+const title = document.getElementById("title");
+const companyDataURL = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/${symbol}`
+const stockDataURL = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/historical-price-full/${symbol}?serietype=line`
+
+addEventListener("load", addCompanyData());
+
+/*async function fetchData (url) {
+    startLoader()
+    const response = await fetch(url);
+    stopLoader()
+    if (response.ok) {
+        const responseJson = await response.json();      
+        return(responseJson);
+        }
+     else {
+        const errorMessage = await response.text();
+        alert(errorMessage);
+    }
+  }*/
+
+async function addCompanyData(){
+    const responseJson = await fetchData (companyDataURL);
+
+    const changeInStock = Number(responseJson.profile.changesPercentage).toFixed(2)
+    if (changeInStock < 0) stockPriceChange.style.color    = "red"
+
+    companyFullName.innerHTML = responseJson.profile.companyName;
+    title.innerHTML = responseJson.profile.companyName + " Information";
+    companyIndustry.innerHTML =responseJson.profile.industry;
+    stockPrice.innerHTML = "$" + responseJson.profile.price + "&nbsp";
+    stockPriceChange.innerHTML = "(" + changeInStock + "%)";
+    companyDescription.innerHTML = responseJson.profile.description;
+    companyLogo.setAttribute("src", responseJson.profile.image);
```

Comment:
If you first store responseJson.profile in a const and then call that variable it is more readable hear:

const profile = responseJson.profile
title.innerHTML = profile.companyName + " Information";